migrate_game.rb

```diff
@@ -263,8 +268,11 @@ def migrate_data(data)
     puts 'Failed to fix :(', e
     return data
   end
-  fixed = true
-  return data if fixed
+
+  # running a migration on a game without issues returns nil actions
+  return unless data['actions']
```

Comment:
This section of the migration seemed very curious to me. Why set fixed to true then immediately use it in the if?

What I found was that if a game did not need a migration and the migrations was run against it, the actions would be set to nil. The new code prevents that from happening and we get the `Nothing to do, game works` as expected.

Now, I am concerned because obviously people are using migrations already, so I am surprised this "bug" would exist, but i doubt its PEBKAC either?